fdb-relational-core/src/test/java/com/apple/foundationdb/relational/recordlayer/query/GroupByQueryTests.java

```diff
@@ -46,6 +47,38 @@ public GroupByQueryTests() {
         Utils.enableCascadesDebugger();
     }
 
+    @Test
+    void isNullPredicateUsesGroupIndex() throws Exception {
```

Comment:
Thank you for writing a proper test case so we won't have a problem like this the next time around!